doc/man/8/rbd.rst

```diff
@@ -358,6 +358,13 @@ Commands
   Release a lock on an image. The lock id and locker are
   as output by lock ls.
 
+:command:`bench` [--io-type read | write] [--io-size *size-in-B/K/M/G/T*] [--io-threads *num-ios-in-flight*] [--io-total *total-size-for-IO-in-B/K/M/G/T*] [--io-pattern seq | rand] *image-spec*
```

Comment:
Minor: remove the "[" and "]" around [--io-type read | write] to imply that it is required.  The automatically generated help text already does this.